src/AssemblerHttpServer-ReadGraph.cpp

```diff
@@ -31,18 +31,53 @@ void Assembler::exploreReadGraph(
 }
 
 
+bool parseCommaSeparatedReadIDs(string& commaSeparatedReadIDs, vector<OrientedReadId>& readIds, ostream& html){
```

Comment:
Shasta uses naming conventions where, in a name containing an acronym, only the first letter is capitalized. So in the name of this function and the first argument `ID` should become `Id`. This is also consistent with `OrientedReadId` and `readIds`.

Reply:
The same comment also in a few more places below.